src/App.js

```diff
@@ -14,11 +16,103 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  addItem = (item) => {
+    this.setState(state => ({
+      selectedGoods: [...state.selectedGoods, item],
+    }));
+  }
+
+  remoteItem = (item) => {
+    this.setState(state => ({
+      selectedGoods: [...state.selectedGoods].filter(el => el !== item),
+    }));
+  }
+
+  remoteAll = () => {
+    this.setState({
+      selectedGoods: [],
+    });
+  }
+
+  createTitle = (listLength) => {
+    if (listLength === 0) {
+      return 'No goods selected';
+    }
+
+    if (listLength === 1) {
+      return `${this.state.selectedGoods[0]} is selected`;
+    }
+
+    return `${this.state.selectedGoods.join(', ')} are selected`;
+  }
+
+  render() {
+    const { selectedGoods } = this.state;
+
+    return (
+      <div className="App d-flex flex-column align-items-center">
+        <div className="App card-body w-75 p-3">
+          <div className="d-flex justify-content-center card-title">
+            <h1 className="px-5">
+              {this.createTitle(selectedGoods.length)}
+            </h1>
+
+            {selectedGoods.length
+              ? (
+                <button
+                  type="button"
+                  className="btn btn-warning clearbtn "
+                  onClick={this.remoteAll}
+                >
+                  X
+                </button>
+              )
+              : ''
+          }
```

Comment:
```suggestion
            {selectedGoods.length && (
                <button
                  type="button"
                  className="btn btn-warning clearbtn "
                  onClick={this.remoteAll}
                >
                  X
                </button>
              )
          }
```
Let's use such conditional rendering syntax